tasks/push-snapshot/tests/test-push-snapshot-mount-certs.yaml

```diff
@@ -0,0 +1,125 @@
+---
+apiVersion: tekton.dev/v1
+kind: Pipeline
+metadata:
+  name: test-push-snapshot
```

Comment:
This should be `test-push-snapshot-mounted-certs`

Reply:
done

Reply:
Sorry... you did what I said but I said the wrong thing :sweat_smile: 
It should be `test-push-snapshot-mount-certs`. The point I am getting at is the test names should be the same as the file name without the `.yaml` extension. The CI would actually timeout if this wasn't the case before

Reply:
done